src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -34,7 +35,11 @@ const ItemUser = ({photo, username, followed, onPress, userid}) => {
         style={[
           styles.containerCard,
           {
-            backgroundColor: isSelect ? COLORS.holytosca15percent : COLORS.white
+            backgroundColor: isSelect
+              ? isAnon
+                ? COLORS.anon_primary + backgroundOpacity15PercentHexCode
+                : COLORS.signed_primary + backgroundOpacity15PercentHexCode
+              : COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color of the container card is now being set conditionally based on the `isSelect` and `isAnon` props. If `isSelect` is true, then the background color will be either `COLORS.anon_primary + backgroundOpacity15PercentHexCode` or `COLORS.signed_primary + backgroundOpacity15PercentHexCode`, depending on the value of `isAnon`. If `isSelect` is false, then the background color will be `COLORS.white`. This change seems to be correct and doesn't introduce any issues.
```diff
- backgroundColor: isSelect ? COLORS.holytosca15percent : COLORS.white
+ backgroundColor: isSelect
+   ? isAnon
+     ? COLORS.anon_primary + backgroundOpacity15PercentHexCode
+     : COLORS.signed_primary + backgroundOpacity15PercentHexCode
+   : COLORS.white
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->